Project.toml

```diff
@@ -20,6 +20,12 @@ SparseArrays = "2f01184e-e22b-5df5-ae63-d93ebab69eaf"
 StaticArrays = "90137ffa-7385-5640-81b9-e52037218182"
 WriteVTK = "64499a7a-5c06-52f2-abe2-ccb03c286192"
 
+[weakdeps]
+Makie = "ee78f7c6-11fb-53f2-987a-cfe4a2b5a57a"
```

Comment:
Please check if we also need to add a version number for Makie in the [compat] section. For regular dependences is needed, but I do not know for weak dependencies.

Reply:
Thanks @fverdugo. I looked into it; it seems that we can add if we want. see, e.g., the discussion in https://github.com/JuliaLang/Pkg.jl/issues/3125. However, I suspect that we may quickly run into compatibility issues since `GLMakie` version (and the `Makie` version coming with it) are installed independent of the package. It may also be the case that other backends `CairoMakie` may come with different `Makie` versions (although I suspect that here we will not be using  `CairoMakie` for 3Ds). I would keep this subject open for the moment, unless you think it is worth investigate it more.

Reply:
Thanks! This is a very good resource. I will add then.